packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.ts

```diff
@@ -51,6 +51,14 @@ export const useAccordionHeader = (
   const size = useContextSelector(AccordionContext, ctx => ctx.size);
   const id = useId('accordion-header-', props.id);
   const panel = useDescendants(accordionItemDescendantContext)[1] as AccordionItemDescendant | undefined;
+  const handleKeyDown = React.useCallback(
+    (ev: React.KeyboardEvent) => {
+      if (ev.key === ' ' || ev.key === 'Enter') {
+        onAccordionHeaderClick(ev);
```

Comment:
```suggestion
        onAccordionHeaderClick(ev);
        state.button.onKeyDown(ev)
```

We also should call an original handler to ensure that user's callbacks will still work. Please add a unit test for this case:

```tsx
<AccordionHeader button={{ onKeyDown }} />
```